UnityProject/Assets/Scripts/Players/PlayerManager.cs

```diff
@@ -66,7 +66,13 @@ public bool CreatePlayer(PlayerDTO playerDTO)
 
             activePlayers.Add(PlayerId(playerDTO.id), player);
 
-			//TODO: Random texture?
+			//TODO: Random texture test
```

Comment:
To be deleted asap after, because we dont want to reload all materials every time. fix indentation